packages/web-components/src/components/expressive-modal/expressive-modal.ts

```diff
@@ -484,10 +484,10 @@ class DDSExpressiveModal extends StableSelectorMixin(
         this.ownerDocument.body.style.overflow = 'hidden';
         this.removeAttribute('aria-hidden');
         this._launcher = this.ownerDocument!.activeElement;
+        await this._waitForTransitionEnd();
```

Comment:
Moving this up before the `querySelector` gives elements time to render into the DOM.